source/connection-monitoring-and-pooling/tests/logging/connection-logging.yml

```diff
@@ -222,3 +222,236 @@ tests:
               reason: "An error occurred while trying to establish a new connection"
               error: { $$exists: true }
               durationMS: { $$type: [double, int, long] }
+
+  - description: "force a pending response read, fail first try, succeed second try"
+    operations:
+      - name: createEntities
```

Comment:
If possible, can we add a test that demonstrates that when the pending read checkout has no timeoutMS set, we use socket_timeout_ms (if it is <3s)?